stdlib/public/core/DoubleWidth.swift.gyb

```diff
@@ -10,21 +10,22 @@
 //
 //===----------------------------------------------------------------------===//
 
-/// A fixed-width integer that is twice the size of its base type.
+/// A fixed-width integer that has twice the bit width of its base type.
 ///
 /// You can use the `DoubleWidth` type to continue calculations with the result
 /// of a full width arithmetic operation. Normally, when you perform a full
-/// width operation, the result is a tuple of the high and low components of
-/// the result.
+/// width operation, the result is a tuple of the high and low parts of the
```

Comment:
In the documentation for integer protocols, "parts" seems to be the preferred term. I think that wording is superior in this case, as these aren't _components_ in the mathematical sense.